cli/src/stake.rs

```diff
@@ -487,6 +500,13 @@ impl StakeSubCommands for App<'_, '_> {
                         .help("Seed for address generation; if specified, the resulting account \
                                will be at a derived address of SPLIT_STAKE_ACCOUNT")
                 )
+                .arg(
+                    Arg::with_name("force")
+                        .long("force")
+                        .takes_value(false)
+                        .hidden(hidden_unless_forced()) // Don't document this argument to discourage its use
```

Comment:
This shouldn't be hidden, right?
```suggestion
```

Reply:
This one should be changed to `skip_balance_checks`, and that flag is not marked as hidden in the other parts where it shows up. I think it's totally fine to keep it visible, since there's no risk if someone uses it accidentally -- they'll just get a worse error from the chain.